spec/views/catalog/_show_tools.html.erb_spec.rb

```diff
@@ -15,7 +15,11 @@
     let(:document_actions) { blacklight_config.show.document_actions }
 
     it "renders a document action" do
-      allow(view).to receive(:some_action_solr_document_path).with(document).and_return "x"
+      if Rails.version < '6.0.0'
+        allow(view).to receive(:some_action_solr_document_path).with(document).and_return "x"
+      else
+        allow(view).to receive(:some_action_solr_document_path).with(document, only_path: true).and_return "x"
```

Comment:
Is it important to strictly test the argument `only_path: true`?  If not, then I think these two cases could be condensed into `allow(view).to receive(:some_action_solr_document_path).with(document, any_args).and_return "x"`

Reply:
I think your suggestion for `any_args` is good. I'll use that.